src/Uno.UI/UI/Xaml/Media/Animation/ObjectAnimationUsingKeyFrames.cs

```diff
@@ -253,8 +253,21 @@ private void Play()
 							CoreDispatcherPriority.Normal,
 							async ct =>
 							{
-								await Task.Delay(dueTime, ct);
-								update();
+								// TODO: Ensure the exception is handled in debug to avoid
+								// breaking here in Visual Studio.
+#if DEBUG
+								try
+								{
+#endif
```

Comment:
It probably would make sense to use `Debugger.IsAttached` instead (or a cache of it) particularly when using Source Link